scalastyle-config.xml

```diff
@@ -94,7 +94,7 @@ This file is divided into 3 sections:
   </check>
 
   <check customId="argcount" level="error" class="org.scalastyle.scalariform.ParameterNumberChecker" enabled="true">
-    <parameters><parameter name="maxParameters"><![CDATA[10]]></parameter></parameters>
```

Comment:
This setting is same with spark, if you change it, it will be hard work to merge to spark in the future.
If you really need more than 10 parameters, please use following instead. More prefer solutions is to obey the code style
```
// scalastyle off
// scalastyle on
```

Reply:
Thanks for your suggestions! Added `scalastyle off and on` for the parameters list.